README.md

```diff
@@ -1,12 +1,10 @@
 # Accessible autocomplete
 
----
-
-**We cannot currently respond to issues or pull requests.**
+The accessible autocomplete is a component that helps users choose answers from a list we provide. You can also use it to standardise user input.
```

Comment:
Do we need to give an example of how it helps standardise input, or is that something our users will understand?